Dockerfile

```
@@ -0,0 +1,11 @@
+FROM openjdk:12-oraclelinux7
+RUN git clone https://github.com/schibsted/jslt.git
+CMD /jslt/gradlew clean shadowJar
+
+
+FROM openjdk:12-oraclelinux7
+COPY ./bin/run_schema_transform.sh /src/
+COPY ./jslt/core/build/libs/core-0.1.11-all.jar /app/run-jslt.jar
+RUN yum update -y && yum install jq -y
+CMD /src/run_schema_transform.sh
```

```suggestion
FROM openjdk:12-oraclelinux7 as build
RUN yum update -y && yum install -y git
RUN git clone https://github.com/schibsted/jslt.git

WORKDIR jslt
RUN /jslt/gradlew clean shadowJar

FROM adoptopenjdk:15-jre
COPY --from=build /jslt/core/build/libs/core-0.1.11-all.jar /app/run-jslt.jar
COPY ./bin/run_schema_transform.sh /src/
RUN apt-get update -q && apt-get install -y jq
```

Can you try with this instead ^^ ? Feel free to change the build image to adoptopenjdk JDK 15 something.

---

Note the --from=build and the lighter image being used in the second from.